test/e2e/storage/external/external.go

```diff
@@ -302,6 +312,42 @@ func (d *driverDefinition) GetDynamicProvisionStorageClass(e2econfig *testsuites
 	return testsuites.GetStorageClass(sc.Provisioner, sc.Parameters, sc.VolumeBindingMode, f.Namespace.Name, "e2e-sc")
 }
 
+func (d *driverDefinition) GetTimeouts() *framework.TimeoutContext {
+	timeouts := framework.NewTimeoutContextWithDefaults()
+	if d.Timeouts == nil {
+		return timeouts
+	}
+
+	// Use a temporary map to hold the timeouts specified in the manifest file
+	c := make(map[string]time.Duration)
```

Comment:
The map should be created with default values. And overwrite the values for the keys provided in the driver config.

Reply:
The `timeouts` struct above contains the default timeouts. This map only contains the custom values provided by the driver config.

Note that down below map is then merged with the struct, overwriting the default values with the custom ones.

Reply:
Ah I see. thanks for pointing it out. missed it.